community/kernel/src/test/java/org/neo4j/kernel/impl/api/index/IndexingServiceTest.java

```diff
@@ -514,69 +519,34 @@ public void shouldLogTriggerSamplingOnAnIndexes() throws Exception
     }
 
     @Test
-    public void recordingOfRecoveredNodesShouldThrowIfServiceIsStarted() throws Exception
```

Comment:
Because there's no way to inject "recovery node ids" at any point in time. That is decided by the lifecycle state now and pulled from the `IndexUpdates` given to `apply`.